[FEATURE] Renommer puis supprimer d'airtable les colonnes traduisibles des épreuves (PIX-9317)

nlepage opened the pull request:
## :unicorn: Problème
Les anciens champs airtable correspondants aux colonnes traduisibles des épreuves sont toujours lues et écrites.

## :robot: Solution
Il faut arrêter de lire et écrire dans ces colonnes maintenant, ça suffit.

## :rainbow: Remarques
N/A

## :100: Pour tester
Tester un peu tout, il y a beaucoup de risques de régression quand même là.

nlepage committed:
Unmap old airtable challenge fields

Co-authored-by: Alix Deschamps <alix.deschamps@pix.fr>
Co-authored-by: Ce que tu veux <guillaume.lagorce@pix.fr>
Co-authored-by: Iris Benoît <iris.benoit@pix.fr>

pix-bot-github commented:
Une fois l'application déployée, elle sera accessible à cette adresse https://pix-lcms-review-pr340.osc-fr1.scalingo.io
Les variables d'environnement seront accessibles sur scalingo https://dashboard.scalingo.com/apps/osc-fr1/pix-lcms-review-pr340/environment

nlepage committed:
fix: challenges' translations migration script must not rely on datasource code

nlepage committed:
refactor: Challenge repository overrides all translatable fields

This is helpful to detect wrongful tests (false positive due to values coming from airtable datasource)

nlepage committed:
fix: static course repository should read challenge instructions from translations

Co-authored-by: Alix Deschamps <alix.deschamps@pix.fr>
Co-authored-by: Ce que tu veux <guillaume.lagorce@pix.fr>

nlepage committed:
refactor: replace old fields by getters in challenge model

Co-authored-by: Iris Benoît <iris.benoit@pix.fr>

Eithliu committed:
refactor: retrieve primary locale in Challenge Model

Co-authored-by: Nicolas Lepage <nicolas.lepage@pix.fr>

nlepage committed:
refactor: all challenge repository methods return domain objects

Co-authored-by: Iris Benoît <iris.benoit@pix.fr>

francois2metz commented:
J'ai comparer 2 release: sur cette RA et une autre release depuis dev, pas de changement.

nlepage reviewed (commented):
LGTM

Eithliu reviewed (approved):
👻 🎃 🍂 LGTM 🍂 🎃 👻
